src/schemas/job/JobRequest.yaml

```diff
@@ -132,3 +132,7 @@ allOf:
                   For example: `* * * * *` represent the cron to launch the job every minute.  
                   See https://crontab.guru/ to WISIWIG interface.  
                   Timezone is UTC
+      auto_deploy:
+        type: boolean
+        nullable: true
+        description: Specify if the job would be automatically updated after receiving a new image or a new commit according to the source type.
```

Comment:
Specify if the job will be automatically updated after receiving a new image tag or a new commit according to the source type. The new image tag shall be communicated via the "Auto Deploy container" endpoint https://api-doc.qovery.com/#tag/Containers/operation/autoDeployContainerEnvironments

Reply:
does the auto deploy container endpoint is also working for job?

Reply:
damn we didn't implement the same endpoint for jobs 🤦 ? I'll open a ticket, so for now on Jobs it will just work for commit

Reply:
done + add Auto Deploy Jobs